articles/confidential-computing/quick-create-confidential-vm-amd-from-azurecomputegalleryimage.md

```diff
@@ -0,0 +1,205 @@
+    --location westus
+    --name mynewCVM \
+    --size “Standard_DC2as_v5” \
+    --image "/subscriptions/00000000-0000-0000-0000-00000000xxxx/resourceGroups/MyResourceGroup/providers/Microsoft.Compute/galleries/MyGallery/images/MyCVMImageDef" \
+    --security-type ConfidentialVM \
+    --os-disk-security-encryption-type DiskwithVMGuestState \
+    --os-disk-securevm-disk-encryption-set $cvmdesId \
+    --enable-secure-boot true \ 
+    --enable-vtpm true \
+    --admin-username $adminUsername \
+    --generate-ssh-keys
+ ```
+
+## Security Type on Image Definition: ConfidentialVMSupported
+
+This security type should be used when the source of the image is an OS disk VHD (without VM Guest state) or a Gen2 managed image.
+The image will not have VM Guest state information and can be used to create either Azure Gen2 VMs or Confidential VMs.
+
+
+### [Portal](#tab/portal2)
+
+1.	Sign in to the [portal](https://portal.azure.com)
+2.	Search for **VM Image Versions** and select **Create**
+3.	Provide the subscription, resource group, region and image version number
+4.	Select the source as either **Storage Blobs (VHD)** or **Managed Image**
+5.	If **Storage Blobs (VHD)** is selected, provide an OS disk VHD (without VM Guest state) as input. Ensure that the OS Disk VHD is a Gen2 VHD.
+6.	If **Managed Image** is selected, provide an existing managed image of an Azure Gen2 VM as input.
+7.	Select a **Target Azure Compute Gallery** to create and share the image. If no gallery exists, create a new gallery.
+9.	Select the **Operating system state** as either **Generalized** or **Specialized**. If VHD is chosen as the source and the OS state is set to generalized, ensure that the VHD is generalized to remove machine specific information before uploading to Azure. If Managed Image is chosen as the source, the operating system state should always be set to Generalized.   
+10.	For the **Target VM Image Definition** select Create new. In the window that opens, select an image definition name and ensure that the **Security type** is set to **Confidential supported**. Provide the publisher, offer and SKU information and select **OK**.
+11.	The **Replication** tab can be used to set the replica count and target regions for image replication, if required.
+12.	The **Encryption** tab can also be used to provide SSE encryption related information. if required.
+13.	Select **Create** in the **Review + create** tab to create the image
+14.	Once the image version is successfully created, select the **+ Create VM** to land on the Create a virtual machine page.
+14.	In the Create a virtual machine page, under **Resource group**, select **Create new** and type a name for your resource group or select an existing resource group from the dropdown.
+15.	Under **Instance details**, type a name for the virtual machine name and choose a region that supports Confidential virtual machines, if Confidential VMs need to be created. If not, any region can be chosen.
```

Comment:
step #14 to #15 is to show customer how to create CVM from ConfidentialVMSupported SIG image, while it is not in above section of ConfidentialVM SIG image. should we add too in above section?